rackcity/views/asset_views.py

```diff
@@ -78,6 +79,26 @@ def asset_many(request):
     assets are returned. If page is specified as a query parameter, page
     size must also be specified, and a page of assets will be returned.
     """
+    if 'change_plan' in request.data:
+        change_plan_id = request.data['change_plan']
+        try:
```

Comment:
not sure if you have strong thoughts about this, but for create/modify endpoints I have been assuming change_plan would be a query param, not in the request body

Reply:
I don't feel strongly and I'd like them to match so I'll make it a query param here